VuChungDung/style.css

```diff
@@ -0,0 +1,122 @@
+*{
+    margin:  0;
+    padding: 0;
+    box-sizing: border-box;
+    font-family: Arial, Helvetica, sans-serif;
+    text-decoration: none;
+}
+.wrap{
+    width: 1170px;
+    margin: 0 auto;
+}
+.header{
+    border-bottom: 1px dotted black;
+}
+.header-content{
+    height: 60px;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+}
+.navbar-menu{
+    display: flex;
+}
+.navbar-menu li{
+    list-style: none;
+    width: 60px;
+    text-align: center;
+    border-right: 1px solid black;
+}
+.navbar-menu__item:last-child{
+    border: none;
+}
+.navbar-menu__link{
+    color: black;
+}
+/* ----------------------------------------------------- */
+.main-content{
+    padding-bottom: 20px;
+    border-bottom: 1px dotted black;
+}
+.main-title{
+    color: dodgerblue;
+    font-size: 74px;
+    margin: 20px 0px;
+}
+.main-detail{
+    display: flex;
+}
+.main-detail__img{
+    flex: 1;
+}
+.main-detail__text{
+    flex: 1;
+}
+.main-detail__img--left{
+    width: 100%;
+}
+.main-detail__text{
+    margin-left: 30px;
+}
+.main-detail__title{
+    padding-bottom: 10px;
+}
+.main-detail__description{
+    line-height: 25px;
+    padding-bottom: 16px;
+}
+.main-detail__list{
+    padding-left: 18px;
+    line-height: 25px;
+    padding-bottom: 30px;
+}
+.main-detail__link{
+    color: black;
+    font-weight: 600;
+}
+.main-detail__footer{
+    display: flex;
+    align-items: center;
+}
+.main-detail__footer--text{
+    color: rgba(255, 102, 0, 0.884);
```

Comment:
I prefer use Hex color code to decimal color code.
Fix for whole page.

Reply:
thanks for reviewing